client/src/views/Record.vue

```diff
@@ -1,95 +1,135 @@
 <template>
-      <div class="bx--grid bx--grid--default bx--grid--no-gutter tell-me-content">
-        <div class="bx--row">
-          <div class="bx--col r1-col">
-            <div class="pagetitle">Tell My Story</div>
-          </div>
-        </div>
-        <div class="bx--row">
-          <div class="bx--col r2-col">
-            <div class="videoRec text-xs-center ">
-              <input type="hidden" ref="video_h" name="video" v-model="videoModel" />
-              <video ref="video"
-              class="video" :poster="poster" controls></video>
-              <div class="video-controllers"></div>
-              <!-- <button @click="startRecording('video1')">Record</button> -->
-              <!-- <button @click="stopRecording('video1')">Stop</button> -->
+  <div>
+    <template>
+          <div class="bx--grid bx--grid--default bx--grid--no-gutter tell-me-content">
+            <div class="bx--row">
+              <div class="bx--col r1-col">
+                <div class="pagetitle">Tell My Story</div>
+              </div>
+            </div>
+            <div class="bx--row">
+              <div class="bx--col r2-col">
+                <div class="videoRec text-xs-center ">
+                  <input type="hidden" ref="video_h" name="video" v-model="videoModel" />
+                  <div class="border" v-if="displayVueWebCam">
+                        <vue-web-cam
+                            ref="webcam"
+                            :device-id="deviceId"
+                            width="100%"
+                            @started="onStarted"
+                            @stopped="onStopped"
+                            @error="onError"
+                            @cameras="onCameras"
+                            @camera-change="onCameraChange"
+                        />
+                    </div>
+                  <video ref="video" v-else
+                  class="video" :poster="poster" controls></video>
+                  <div class="video-controllers"></div>
+                  <!-- <button @click="startRecording('video1')">Record</button> -->
+                  <!-- <button @click="stopRecording('video1')">Stop</button> -->
+                </div>
+              </div>
+            </div>
+            <div class="bx--row">
+              <div class="bx--col r3-col">
+                <cv-grid class="video-control-panel">
+                  <cv-row class="grid-row">
+                    <cv-column class="col-1">
+                        <div>Full Detail</div>
+                    </cv-column>
+                    <cv-column class="col-2">
+                        <div class="btn-blur">Blur</div>
+                    </cv-column>
+                    <cv-column class="col-3">
+                        <div>Audio Only</div>
+                    </cv-column>
+                  </cv-row>
+                  <cv-row class="grid-row">
+                    <cv-column class="col-1">
+                      Lulu Blake
```

Comment:
Is the hard-coded "Lulu Blake" here just a placeholder?

Reply:
Hi, that was already there, I was wondering the same thing. So, we just left it how it was.

Reply:
thanks for the quick reply.  I'll merge this